### ISSUE TITLE
Telepresence connect returns telepresence connect: error: connector.Connect: failed to connect to root daemon: rpc error: code = DeadlineExceeded desc = context deadline exceeded

### ISSUE THREAD

**Issue by dfilipov-anakatech** [telepresenceio/telepresence#3350]
When trying to 'telepresence connect' we get the error message 
'telepresence connect: error: connector.Connect: failed to connect to root daemon: rpc error: code = DeadlineExceeded desc = context deadline exceeded'
None of the telepresence functionality is thus available.
Running 'telepresence loglevel debug' leads to the same exact error, so I cannot provide more verbose logs than the default.
We are using an OpenVPN server and client and this problem only occurs on Windows machines

connector.log
```2023-09-20 11:26:07.2711 info    Starting socket listener for C:\Users\Dimba\AppData\Local\telepresence\userd.socket
2023-09-20 11:26:07.2716 info    ---
2023-09-20 11:26:07.2716 info    Telepresence Connector v2.15.1 (api v3) starting...
2023-09-20 11:26:07.2716 info    PID is 4532
2023-09-20 11:26:07.2716 info    
2023-09-20 11:26:07.4207 info    connector/server-grpc : gRPC server started
2023-09-20 11:26:07.7501 info    connector/session : -- Starting new session
2023-09-20 11:26:07.7501 info    connector/session : Connecting to k8s cluster...
2023-09-20 11:26:07.9992 info    connector/session : Server version v1.27.5-gke.200
2023-09-20 11:26:07.9992 info    connector/session : Context: gke_netomedia2_europe-west1_qa-cluster
2023-09-20 11:26:07.9992 info    connector/session : Server: https://<our API server ip>
2023-09-20 11:26:11.1442 info    connector/session : Will look for traffic manager in namespace ambassador
2023-09-20 11:26:11.1442 info    connector/session : Connected to context gke_netomedia2_europe-west1_qa-cluster (https://35.195.249.210)
2023-09-20 11:26:11.2108 info    connector/session : Connecting to traffic manager...
2023-09-20 11:26:11.7604 info    connector/session : Connected to traffic-manager v2.14.2
2023-09-20 11:26:11.8534 info    connector/session : Configuration reloaded
2023-09-20 11:26:11.8550 info    connector/session : Connecting to root daemon...
2023-09-20 11:28:52.3343 info    connector/session : -- Starting new session
2023-09-20 11:28:52.3343 info    connector/session : Connecting to k8s cluster...
2023-09-20 11:28:52.4938 info    connector/session : Server version v1.27.5-gke.200
2023-09-20 11:28:52.4938 info    connector/session : Context: gke_netomedia2_europe-west1_qa-cluster
2023-09-20 11:28:52.4938 info    connector/session : Server: https://<our API server ip>
2023-09-20 11:28:55.6324 info    connector/session : Will look for traffic manager in namespace ambassador
2023-09-20 11:28:55.6324 info    connector/session : Connected to context gke_netomedia2_europe-west1_qa-cluster (https://35.195.249.210)
2023-09-20 11:28:55.7003 info    connector/session : Connecting to traffic manager...
2023-09-20 11:28:56.2296 info    connector/session : Connected to traffic-manager v2.14.2
2023-09-20 11:28:56.3243 info    connector/session : Configuration reloaded
2023-09-20 11:28:56.3270 info    connector/session : Connecting to root daemon...
2023-09-20 11:29:14.9349 error   connector/server-grpc/conn=4 : Tunnel manager.Recv() failed: rpc error: code = Canceled desc = grpc: the client connection is closing
2023-09-20 11:29:14.9349 error   connector/server-grpc/conn=4 : Tunnel manager.Recv() failed: rpc error: code = Canceled desc = grpc: the client connection is closing
2023-09-20 11:29:14.9349 error   connector/server-grpc/conn=4 : Tunnel manager.Recv() failed: rpc error: code = Canceled desc = grpc: the client connection is closing
2023-09-20 11:29:14.9349 error   connector/server-grpc/conn=4 : Tunnel manager.Recv() failed: rpc error: code = Canceled desc = grpc: the client connection is closing
2023-09-20 11:29:14.9349 error   connector/server-grpc/conn=4 : Tunnel manager.Recv() failed: rpc error: code = Canceled desc = grpc: the client connection is closing
2023-09-20 11:29:14.9349 error   connector/server-grpc/conn=4 : Tunnel manager.Recv() failed: rpc error: code = Canceled desc = grpc: the client connection is closing
2023-09-20 11:29:14.9349 error   connector/server-grpc/conn=4 : Tunnel manager.Recv() failed: rpc error: code = Canceled desc = grpc: the client connection is closing
2023-09-20 11:29:14.9349 error   connector/server-grpc/conn=4 : Tunnel manager.Recv() failed: rpc error: code = Canceled desc = grpc: the client connection is closing
2023-09-20 11:29:14.9349 error   connector/server-grpc/conn=4 : Tunnel manager.Recv() failed: rpc error: code = Canceled desc = grpc: the client connection is closing
2023-09-20 11:29:14.9349 error   connector/server-grpc/conn=4 : Tunnel manager.Recv() failed: rpc error: code = Canceled desc = grpc: the client connection is closing
2023-09-20 11:29:14.9349 error   connector/server-grpc/conn=4 : Tunnel manager.Recv() failed: rpc error: code = Canceled desc = grpc: the client connection is closing
2023-09-20 11:29:14.9359 error   connector/server-grpc/conn=4 : Tunnel manager.Recv() failed: rpc error: code = Canceled desc = grpc: the client connection is closing
2023-09-20 11:29:14.9359 error   connector/server-grpc/conn=4 : Tunnel manager.Recv() failed: rpc error: code = Canceled desc = grpc: the client connection is closing
2023-09-20 11:29:14.9359 error   connector/server-grpc/conn=4 : Tunnel manager.Recv() failed: rpc error: code = Canceled desc = grpc: the client connection is closing
2023-09-20 11:29:14.9359 error   connector/server-grpc/conn=4 : Tunnel manager.Recv() failed: rpc error: code = Canceled desc = grpc: the client connection is closing
2023-09-20 11:29:14.9359 error   connector/server-grpc/conn=4 : Tunnel manager.Recv() failed: rpc error: code = Canceled desc = grpc: the client connection is closing
2023-09-20 11:29:14.9359 error   connector/server-grpc/conn=4 : Tunnel manager.Recv() failed: rpc error: code = Canceled desc = grpc: the client connection is closing
2023-09-20 11:29:14.9359 error   connector/server-grpc/conn=4 : Tunnel manager.Recv() failed: rpc error: code = Canceled desc = grpc: the client connection is closing
2023-09-20 11:29:14.9359 error   connector/server-grpc/conn=4 : Tunnel manager.Recv() failed: rpc error: code = Canceled desc = grpc: the client connection is closing
2023-09-20 11:29:14.9359 error   connector/server-grpc/conn=4 : Tunnel manager.Recv() failed: rpc error: code = Canceled desc = grpc: the client connection is closing
2023-09-20 11:29:14.9359 error   connector/server-grpc/conn=4 : Tunnel manager.Recv() failed: rpc error: code = Canceled desc = grpc: the client connection is closing
2023-09-20 11:29:14.9359 error   connector/server-grpc/conn=4 : Tunnel manager.Recv() failed: rpc error: code = Canceled desc = grpc: the client connection is closing
2023-09-20 11:29:14.9359 error   connector/server-grpc/conn=4 : Tunnel manager.Recv() failed: rpc error: code = Canceled desc = grpc: the client connection is closing
2023-09-20 11:29:14.9359 error   connector/server-grpc/conn=4 : Tunnel manager.Recv() failed: rpc error: code = Canceled desc = grpc: the client connection is closing
2023-09-20 11:29:14.9359 error   connector/server-grpc/conn=4 : Tunnel manager.Recv() failed: rpc error: code = Canceled desc = grpc: the client connection is closing
2023-09-20 11:29:14.9359 error   connector/server-grpc/conn=4 : Tunnel manager.Recv() failed: rpc error: code = Canceled desc = grpc: the client connection is closing```



* reproduce the error *
Run' telepresence connect'

**To Reproduce**
Steps to reproduce the behavior:
1. When I run 'telepresence connect'
2. I see 'telepresence loglevel: error: connector.Connect: failed to connect to root daemon: rpc error: code = DeadlineExceeded desc = context deadline exceeded'

daemon.log

`2023-09-20 11:26:07.5380 info    ---
2023-09-20 11:26:07.5380 info    Telepresence daemon v2.15.1 (api v3) starting...
2023-09-20 11:26:07.5380 info    PID is 15136
2023-09-20 11:26:07.5380 info    
2023-09-20 11:26:07.6220 info    daemon/server-grpc : gRPC server started
2023-09-20 11:26:11.8577 info    daemon/session : -- Starting new session
2023-09-20 11:26:11.9027 info    daemon/session : Connected to Manager 2.14.2
2023-09-20 11:26:11.9471 info    daemon/session : also-proxy subnets []
2023-09-20 11:26:11.9471 info    daemon/session : never-proxy subnets <our API server ip>/32]
2023-09-20 11:26:11.9471 info    daemon/session : Configuration reloaded
2023-09-20 11:26:12.9969 info    daemon/session/network : also-proxy subnets []
2023-09-20 11:26:12.9969 info    daemon/session/network : never-proxy subnets [<our API server ip>/32]
2023-09-20 11:26:13.0005 info    daemon/session/network : Creating interface tel0
2023-09-20 11:26:13.0718 info    stdlog : Using existing driver 0.14
2023-09-20 11:26:13.0785 info    stdlog : Creating adapter
2023-09-20 11:26:13.2327 info    daemon/session/network : also-proxy subnets []
2023-09-20 11:26:13.2330 info    daemon/session/network : never-proxy subnets [<our API server ip>/32]
2023-09-20 11:26:13.2330 info    daemon/session/network : Adding Service subnet 10.203.0.0/16
2023-09-20 11:26:13.2330 info    daemon/session/network : Adding pod subnet 10.202.0.0/25
2023-09-20 11:26:13.2330 info    daemon/session/network : Adding pod subnet 10.202.0.128/25
2023-09-20 11:26:13.2330 info    daemon/session/network : Adding pod subnet 10.202.1.0/25
2023-09-20 11:26:13.2330 info    daemon/session/network : Adding pod subnet 10.202.1.128/25
2023-09-20 11:26:13.2330 info    daemon/session/network : Adding pod subnet 10.202.2.0/25
2023-09-20 11:26:13.2330 info    daemon/session/network : Adding pod subnet 10.202.2.128/25
2023-09-20 11:26:13.2330 info    daemon/session/network : Adding pod subnet 10.202.3.0/25
2023-09-20 11:26:13.2330 info    daemon/session/network : Adding pod subnet 10.202.3.128/25
2023-09-20 11:26:13.2330 info    daemon/session/network : Adding pod subnet 10.202.4.0/25
2023-09-20 11:26:13.2330 info    daemon/session/network : Adding pod subnet 10.202.4.128/25
2023-09-20 11:26:13.2330 info    daemon/session/network : Adding pod subnet 10.202.5.0/25
2023-09-20 11:26:13.2330 info    daemon/session/network : Adding pod subnet 10.202.5.128/25
2023-09-20 11:26:13.2330 info    daemon/session/network : Adding pod subnet 10.202.6.0/25
2023-09-20 11:26:13.2330 info    daemon/session/network : Adding pod subnet 10.202.6.128/25
2023-09-20 11:26:13.2330 info    daemon/session/network : Adding pod subnet 10.202.7.0/25
2023-09-20 11:26:13.2330 info    daemon/session/network : Adding pod subnet 10.202.7.128/25
2023-09-20 11:26:13.2330 info    daemon/session/network : Adding pod subnet 10.202.8.0/25
2023-09-20 11:26:13.2330 info    daemon/session/network : Adding pod subnet 10.202.8.128/25
2023-09-20 11:26:13.2330 info    daemon/session/network : Adding pod subnet 10.202.9.0/25
2023-09-20 11:26:13.2330 info    daemon/session/network : Adding pod subnet 10.202.9.128/25
2023-09-20 11:26:13.2330 info    daemon/session/network : Adding pod subnet 10.202.10.0/25
2023-09-20 11:26:13.2330 info    daemon/session/network : Adding pod subnet 10.202.10.128/25
2023-09-20 11:26:13.2330 info    daemon/session/network : Adding pod subnet 10.202.11.0/25
2023-09-20 11:26:13.2330 info    daemon/session/network : Adding pod subnet 10.202.11.128/25
2023-09-20 11:26:13.2330 info    daemon/session/network : Adding pod subnet 10.202.12.0/25
2023-09-20 11:26:13.2330 info    daemon/session/network : Setting cluster DNS to 10.202.3.14
2023-09-20 11:26:13.2330 info    daemon/session/network : Setting cluster domain to "cluster.local."
2023-09-20 11:26:13.2375 info    daemon/session/network : Starting Endpoint
2023-09-20 11:26:28.4125 error   daemon/session/dns/RecursionCheck : unexpected error during recursion check: lookup tel2-recursion-check.kube-system.tel2-search: no such host
2023-09-20 11:26:28.5232 error   daemon/session/dns/RecursionCheck : unexpected error during recursion check: lookup tel2-recursion-check.kube-system.tel2-search: no such host
2023-09-20 11:26:28.6330 error   daemon/session/dns/RecursionCheck : unexpected error during recursion check: lookup tel2-recursion-check.kube-system.tel2-search: no such host
2023-09-20 11:26:28.7413 error   daemon/session/dns/RecursionCheck : unexpected error during recursion check: lookup tel2-recursion-check.kube-system.tel2-search: no such host
2023-09-20 11:26:28.8501 error   daemon/session/dns/RecursionCheck : unexpected error during recursion check: lookup tel2-recursion-check.kube-system.tel2-search: no such host
2023-09-20 11:26:28.9587 error   daemon/session/dns/RecursionCheck : unexpected error during recursion check: lookup tel2-recursion-check.kube-system.tel2-search: no such host
2023-09-20 11:26:29.0667 error   daemon/session/dns/RecursionCheck : unexpected error during recursion check: lookup tel2-recursion-check.kube-system.tel2-search: no such host
2023-09-20 11:26:29.1734 error   daemon/session/dns/RecursionCheck : unexpected error during recursion check: lookup tel2-recursion-check.kube-system.tel2-search: no such host
2023-09-20 11:26:29.2846 error   daemon/session/dns/RecursionCheck : unexpected error during recursion check: lookup tel2-recursion-check.kube-system.tel2-search: no such host
2023-09-20 11:26:29.3946 error   daemon/session/dns/RecursionCheck : unexpected error during recursion check: lookup tel2-recursion-check.kube-system.tel2-search: no such host
2023-09-20 11:26:29.5043 error   daemon/session/dns/RecursionCheck : unexpected error during recursion check: lookup tel2-recursion-check.kube-system.tel2-search: no such host
2023-09-20 11:26:29.6113 error   daemon/session/dns/RecursionCheck : unexpected error during recursion check: lookup tel2-recursion-check.kube-system.tel2-search: no such host
2023-09-20 11:26:29.7215 error   daemon/session/dns/RecursionCheck : unexpected error during recursion check: lookup tel2-recursion-check.kube-system.tel2-search: no such host
2023-09-20 11:26:29.8308 error   daemon/session/dns/RecursionCheck : unexpected error during recursion check: lookup tel2-recursion-check.kube-system.tel2-search: no such host
2023-09-20 11:26:29.9405 error   daemon/session/dns/RecursionCheck : unexpected error during recursion check: lookup tel2-recursion-check.kube-system.tel2-search: no such host
2023-09-20 11:26:30.0540 error   daemon/session/dns/RecursionCheck : unexpected error during recursion check: lookup tel2-recursion-check.kube-system.tel2-search: no such host
2023-09-20 11:26:30.1599 error   daemon/session/dns/RecursionCheck : unexpected error during recursion check: lookup tel2-recursion-check.kube-system.tel2-search: no such host
2023-09-20 11:26:30.2697 error   daemon/session/dns/RecursionCheck : unexpected error during recursion check: lookup tel2-recursion-check.kube-system.tel2-search: no such host
2023-09-20 11:26:30.3315 info    daemon/session/dns/Server:shutdown_logger : shutting down (gracefully)...
2023-09-20 11:26:30.3315 error   daemon/session/dns/RecursionCheck : unexpected error during recursion check: lookup tel2-recursion-check.kube-system.tel2-search: operation was canceled
2023-09-20 11:26:30.3315 info    daemon/session/dns:shutdown_logger : shutting down (gracefully)...
2023-09-20 11:26:30.3315 error   daemon/session/dns/RecursionCheck : unexpected error during recursion check: lookup tel2-recursion-check.kube-system.tel2-search: operation was canceled
2023-09-20 11:26:30.3315 error   daemon/session/dns/RecursionCheck : unexpected error during recursion check: lookup tel2-recursion-check.kube-system.tel2-search: operation was canceled
2023-09-20 11:26:30.3315 error   daemon/session/dns/RecursionCheck : unexpected error during recursion check: lookup tel2-recursion-check.kube-system.tel2-search: operation was canceled
2023-09-20 11:26:30.3315 error   daemon/session/dns/RecursionCheck : unexpected error during recursion check: lookup tel2-recursion-check.kube-system.tel2-search: operation was canceled
2023-09-20 11:26:30.3315 error   daemon/session/dns/RecursionCheck : unexpected error during recursion check: lookup tel2-recursion-check.kube-system.tel2-search: operation was canceled
2023-09-20 11:26:30.3315 error   daemon/session/dns/RecursionCheck : unexpected error during recursion check: lookup tel2-recursion-check.kube-system.tel2-search: operation was canceled
2023-09-20 11:26:30.3315 error   daemon/session/dns/RecursionCheck : unexpected error during recursion check: lookup tel2-recursion-check.kube-system.tel2-search: operation was canceled
2023-09-20 11:26:30.3320 error   daemon/session/dns/RecursionCheck : unexpected error during recursion check: lookup tel2-recursion-check.kube-system.tel2-search: operation was canceled
2023-09-20 11:26:30.3320 error   daemon/session/dns/RecursionCheck : unexpected error during recursion check: lookup tel2-recursion-check.kube-system.tel2-search: operation was canceled
2023-09-20 11:26:30.3320 error   daemon/session/dns/RecursionCheck : unexpected error during recursion check: lookup tel2-recursion-check.kube-system.tel2-search: operation was canceled
2023-09-20 11:26:30.3320 error   daemon/session/dns/RecursionCheck : unexpected error during recursion check: lookup tel2-recursion-check.kube-system.tel2-search: operation was canceled
2023-09-20 11:26:30.3320 error   daemon/session/dns/RecursionCheck : unexpected error during recursion check: lookup tel2-recursion-check.kube-system.tel2-search: operation was canceled
2023-09-20 11:26:30.3320 error   daemon/session/dns/RecursionCheck : unexpected error during recursion check: lookup tel2-recursion-check.kube-system.tel2-search: operation was canceled
2023-09-20 11:26:30.3320 error   daemon/session/dns/RecursionCheck : unexpected error during recursion check: lookup tel2-recursion-check.kube-system.tel2-search: operation was canceled
2023-09-20 11:26:31.2589 info    daemon/session/network : Endpoint done
2023-09-20 11:26:33.5733 info    daemon/session : -- Session ended
2023-09-20 11:26:33.5743 info    daemon/session : Configuration reloaded`

**Expected behavior**
Be able to connect to the cluster

**Versions (please complete the following information):**
 - Output of `telepresence version`
 OSS Client     : v2.15.1
 OSS Root Daemon: v2.15.1
 OSS User Daemon: v2.15.1
 Traffic Manager: not connected
 - Operating system of workstation running `telepresence` commands
 Windows 10
 - Kubernetes environment and Version [e.g. Minikube, bare metal, Google Kubernetes Engine]
 GKE 1.27

**VPN-related bugs:**
If you're reporting an issue around telepresence connectivity when using a VPN,
and are running Telepresence 2.4.8 or above, please also attach the output
of `telepresence test-vpn`, and the following information:
 - Which VPN client are you using?
 OpenVPN
 - Which VPN server are you using?
 OpenVPN
 - How is your VPN pushing DNS configuration? It may be useful to add the contents of /etc/resolv.conf
It is using the built-in DHCP server of Windows

**Comment by thallgren** [comment]
This seems to be an issue with DNS. What Windows version are you using?

**Comment by dfilipov-anakatech** [comment]
@thallgren this occurs on both Windows 10 and Windows 11, the exact versions differ a bit, but the problem is the same.
The one I'm using is Windows 10 x64 version 10.0.19045

**Comment by east-shine** [comment]
same issue.(M1 Mac)

```
telepresence version
OSS Client     : v2.17.0
OSS Root Daemon: v2.17.0
OSS User Daemon: v2.17.0
Traffic Manager: not connected
```

```
sudo telepresence connect
Launching Telepresence User Daemon
Launching Telepresence Root Daemon
telepresence connect: error: connector.Connect: failed to connect to root daemon: rpc error: code = DeadlineExceeded desc = context deadline exceeded
```

```
2023-12-30 00:48:18.7156 info    daemon/server-grpc : gRPC server started
2023-12-30 00:48:19.0650 info    daemon/session : -- Starting new session
2023-12-30 00:48:19.1438 info    daemon/session : Connected to OSS Traffic Manager v2.17.0
2023-12-30 00:48:19.1439 info    daemon/session : Connected to Manager 2.17.0
2023-12-30 00:48:19.1459 info    daemon/session : also-proxy subnets []
2023-12-30 00:48:19.1460 info    daemon/session : never-proxy subnets [127.0.0.1/32]
2023-12-30 00:48:19.1463 info    daemon/session : Configuration reloaded
2023-12-30 00:48:19.1528 info    daemon/session/network : Already connected to cluster, will not map pod subnets.
2023-12-30 00:48:19.1562 info    daemon/session/network : Already connected to cluster, will not map service subnets.
2023-12-30 00:48:19.1563 info    daemon/session/network : also-proxy subnets []
2023-12-30 00:48:19.1563 info    daemon/session/network : never-proxy subnets [127.0.0.1/32]
2023-12-30 00:48:19.1564 info    daemon/session/network : also-proxy subnets []
2023-12-30 00:48:19.1564 info    daemon/session/network : never-proxy subnets [127.0.0.1/32]
2023-12-30 00:48:19.1564 info    daemon/session/network : Setting cluster DNS to 192.168.194.100
2023-12-30 00:48:19.1564 info    daemon/session/network : Setting cluster domain to "cluster.local."
2023-12-30 00:48:19.1571 info    daemon/session/dns : Generated new /etc/resolver/telepresence.local
2023-12-30 00:48:19.1574 error   daemon/session/dns/RecursionCheck : goroutine "/daemon/session/dns/RecursionCheck" exited with error: PANIC: runtime error: invalid memory address or nil pointer dereference
github.com/datawire/dlib/dgroup.(*Group).goWorkerCtx.func1.1
	github.com/datawire/dlib@v1.3.1/dgroup/group.go:369
runtime.gopanic
	runtime/panic.go:914
runtime.panicmem
	runtime/panic.go:261
runtime.sigpanic
	runtime/signal_unix.go:861
github.com/telepresenceio/telepresence/v2/pkg/client/rootd/dns.(*Server).processSearchPaths.func1
	github.com/telepresenceio/telepresence/v2/pkg/client/rootd/dns/server.go:426
github.com/datawire/dlib/dgroup.(*Group).goWorkerCtx.func1
	github.com/datawire/dlib@v1.3.1/dgroup/group.go:385
github.com/datawire/dlib/derrgroup.(*Group).Go.func2
	github.com/datawire/dlib@v1.3.1/derrgroup/errgroup.go:132
runtime.goexit
	runtime/asm_arm64.s:1197
2023-12-30 00:48:19.1575 info    daemon/session/dns:shutdown_logger : shutting down (gracefully)...
2023-12-30 00:48:19.1575 info    daemon/session/dns/Server:shutdown_logger : shutting down (gracefully)...
2023-12-30 00:48:19.1576 info    daemon/session/dns:shutdown_status :   final goroutine statuses:
2023-12-30 00:48:19.1577 info    daemon/session/dns:shutdown_status :     /daemon/session/dns/RecursionCheck: exited with error
2023-12-30 00:48:19.1577 info    daemon/session/dns:shutdown_status :     /daemon/session/dns/SearchPaths   : exited
2023-12-30 00:48:19.1577 info    daemon/session/dns:shutdown_status :     /daemon/session/dns/Server        : exited
2023-12-30 00:48:19.1579 error   daemon/session/dns : goroutine "/daemon/session/dns" exited with error: PANIC: runtime error: invalid memory address or nil pointer dereference
github.com/datawire/dlib/dgroup.(*Group).goWorkerCtx.func1.1
	github.com/datawire/dlib@v1.3.1/dgroup/group.go:369
runtime.gopanic
	runtime/panic.go:914
runtime.panicmem
	runtime/panic.go:261
runtime.sigpanic
	runtime/signal_unix.go:861
github.com/telepresenceio/telepresence/v2/pkg/client/rootd/dns.(*Server).processSearchPaths.func1
	github.com/telepresenceio/telepresence/v2/pkg/client/rootd/dns/server.go:426
github.com/datawire/dlib/dgroup.(*Group).goWorkerCtx.func1
	github.com/datawire/dlib@v1.3.1/dgroup/group.go:385
github.com/datawire/dlib/derrgroup.(*Group).Go.func2
	github.com/datawire/dlib@v1.3.1/derrgroup/errgroup.go:132
runtime.goexit
	runtime/asm_arm64.s:1197
2023-12-30 00:48:19.1579 info    daemon/session:shutdown_logger : shutting down (gracefully)...
2023-12-30 00:48:19.1579 error   daemon/session/agentPods : goroutine "/daemon/session/agentPods" exited with error: rpc error: code = Canceled desc = context canceled
2023-12-30 00:48:19.1580 info    daemon/session:shutdown_status :   final goroutine statuses:
2023-12-30 00:48:19.1580 info    daemon/session:shutdown_status :     /daemon/session/agentPods: exited with error
2023-12-30 00:48:19.1580 info    daemon/session:shutdown_status :     /daemon/session/dns      : exited with error
2023-12-30 00:48:19.1580 info    daemon/session:shutdown_status :     /daemon/session/network  : exited
2023-12-30 00:48:19.1580 info    daemon/session : -- Session ended
2023-12-30 00:48:19.1589 error   daemon/session : PANIC: runtime error: invalid memory address or nil pointer dereference
```

**Comment by thallgren** [comment]
@east-shine I don't think this is the same issue. You encounter a panic on a M1. Can you please create a new issue?

**Comment by thallgren** [comment]
The M1 panic (actually, it affects all darwin systems) is addressed in PR #3474.

**Comment by thallgren** [comment]
@dfilipov-anakatech would you care to try with the new 2.19.0 release? It contains some improvements that might impact this.
